framework/validators/Validator.php

```diff
@@ -234,6 +234,34 @@ public function init()
     }
 
     /**
+     * Delegates validation to appropriate method depending on attributes.
+     * @param \yii\base\Model $model the data model being validated.
+     * @param array|null $attributes the list of attributes to be validated.
+     * @since 2.0.11
+     */
+    public function delegateValidation($model, $attributes)
```

Comment:
Yes, but name can be different. Do you think it's better to leave this in `validateAttributes` method (at the very beginning)?